### DISCUSSION TITLE
Project Showcase

### DISCUSSION THREAD

**Discussion by azzamsa** [emilk/egui#974] [Show and tell]
Hi.

I think it is nice to have project showcase in one place such [iced showcase](https://github.com/iced-rs/iced/issues/355) here.

**Comment by tobtobxx** [comment]
Isn't that exactly what this Show and Tell section is for?

**Reply by sourcebox** [reply]
> Isn't that exactly what this Show and Tell section is for?

I wasn't really aware of this section until now. It's really easy to miss and the fact that there are not many posts there seems to prove it. Maybe a link from the readme would help.

**Comment by emilk** [comment]
I think this is a good idea, and I agree the "Show and Tell" section of the Discussions forum isn't that visible. I'll create a pinned issue instead, with some guidelines of what to post.

EDIT: here it is: https://github.com/emilk/egui/issues/996

**Comment by BctfN0HUK7Yg** [comment]
check egui discord , there ara some showcases
https://discord.com/channels/900275882684477440/900284657428492299
